src/Parser/BrewmasterMonk/Modules/Features/AlwaysBeCasting.js

```diff
@@ -35,13 +37,13 @@ class AlwaysBeCasting extends CoreAlwaysBeCastingHealing {
   suggestions(when) {
     const deadTimePercentage = this.totalTimeWasted / this.owner.fightDuration;
 
-    when(deadTimePercentage).isGreaterThan(0.4)
+    when(deadTimePercentage).isGreaterThan(0.2)
       .addSuggestion((suggest, actual, recommended) => {
-        return suggest('Your dead GCD time can be improved. Try to Always Be Casting (ABC); try to reduce the delay between casting spells and when you\'re not healing try to contribute some damage.')
+        return suggest('Your dead GCD time can be improved. Try to Always Be Casting (ABC).')
```

Comment:
Explaining how to do this is pretty important as it's non-obvious to users (they're often unaware of big gaps between spells being cast)